hr_expense_analytic_tag/models/hr_expense.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class HrExpense(models.Model):
+    _inherit = "hr.expense"
+
+    analytic_tag_ids = fields.Many2many(
+        comodel_name='account.analytic.tag',
+        string='Analytic Tags',
+        copy=True
+    )
+
+    def _prepare_move_line(self, line):
+        line_values = super(HrExpense, self)._prepare_move_line(line)
+        if line_values['analytic_account_id']:
+            tag_values = [[4, tag.id] for tag in self.analytic_tag_ids]
```

Comment:
Shouldn't this be `line.analytic_tag_ids`?
```suggestion
            tag_values = [[4, tag.id] for tag in line.analytic_tag_ids]
```

Reply:
line is a dictionary, so it doesn't have analytic_tag_ids attribute, we just want the expense tags and we access this doing that (self.analytic_tag_ids), just like in this method in the super class it get the partner of the expense (partner_id = self.employee_id.address_home_id.commercial_partner_id.id)

Reply:
OK